docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,29 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 20th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using Fivetran and its [Stripe connector](https://fivetran.com/docs/applications/stripe). This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It also includes information regarding subscriptions and customers, as well as products, plans, invoices, cards, charges, discounts, coupons, and promotion codes.
+
+Some of the customer information includes the customer's postal code, state, and country for a subscription, which helps resolve issues for customers across a particular dimension (e.g., country, payment provider, etc.). However, as it intentionally excludes any personal identifiable information of customers (e.g., name, street address, phone number, email address), it can be challenging to use to help on an individual customer level. Other uses for this data is for retrieving all-time counts (e.g., total number of active subscriptions).
+
+Fivetran preserves historical data within `subscription_history` by comparing the current state of the subscription with the most recent record in the history table and adding a new record if they vary.
```

Comment:
IIUC this is limited in terms of what history we can get and/or is difficult to identify what changed between one version of a Stripe subscription object and a later version, which is why we will also rely on a Stripe Firestore sync for historical subscription data. @sean-rose , can you confirm this? It'd be nice to discuss these limitations briefly if so and mention that as of February 2023, we also sync SubPlat's Stripe Firestore Customer and Subscriptions collections (see https://mozilla-hub.atlassian.net/browse/SVCSE-1080).

Reply:
> IIUC this is limited in terms of what history we can get and/or is difficult to identify what changed between one version of a Stripe subscription object and a later version, which is why we will also rely on a Stripe Firestore sync for historical subscription data. @sean-rose , can you confirm this?

The main issue with the Fivetran Stripe sync is it only records historical state/changes for the top-level subscription fields and not the associated subscription items which indicate the subscribed plan, so we have to rely on the subscription metadata fields `previous_plan_id` and `plan_change_date`.  However, those only record the most recent plan change, so if there are any problems with the Fivetran sync (like [happened last month](https://mozilla-hub.atlassian.net/browse/DENG-754)) then we can miss/lose history.

So yes, the current plan is to make use of the Stripe Firestore changelog data, but before actually doing that we'll need to verify that the Stripe Firestore collection is reliable (e.g. this week I found some issues with Firestore missing some subscriptions), and since the Stripe Firestore collection doesn't contain all historical subscriptions/changes we'll still need to use the Fivetran data for some subscriptions.

> It'd be nice to discuss these limitations briefly if so and mention that as of February 2023, we also sync SubPlat's Stripe Firestore Customer and Subscriptions collections (see https://mozilla-hub.atlassian.net/browse/SVCSE-1080).

It'd also be worth noting when the Stripe Firestore collections themself were created.

Reply:
> So yes, the current plan is to make use of the Stripe Firestore changelog data, but before actually doing that we'll need to verify that the Stripe Firestore collection is reliable (e.g. this week I found some issues with Firestore missing some subscriptions), and since the Stripe Firestore collection doesn't contain all historical subscriptions/changes we'll still need to use the Fivetran data for some subscriptions.

Yep we've been discussing this in [Slack](https://mozilla.slack.com/archives/CG0AJ6E77/p1679605292654949). As I mentioned there, these aren't bugs in our implementation, but rather represent a new use case that differs from our original intent/design.

Reply:
> It might also be worth noting when the Stripe Firestore collections themselves were created.

This is October 2021 per [FXA-4023](https://mozilla-hub.atlassian.net/browse/FXA-4023).